akka-actor/src/main/resources/reference.conf

```diff
@@ -99,6 +99,11 @@ akka {
   # by setting this property to `off`.
   fail-mixed-versions = on
 
+  # Some modules (remoting only right now) can emit custom events to the Java Flight Recorder if running
+  # on JDK 11 or later. If you for some reason do not want that, it can be disabled and switched to no-ops
+  # with this toggle.
+  use-java-flight-recorder = on
```

Comment:
```
akka.flight-recorder {
  enabled = on
}
```

to prepare for more settings in the future?

Reply:
I thought of a global on switch as enough, since you can always configure JFR selectively when you have that enabled.

Reply:
Let's do that scope anyway just in case.